contacts.module

```diff
@@ -130,7 +133,40 @@ function template_preprocess_contacts_dash_tabs(array &$variables) {
     ],
   ];
 
-  foreach ($variables['tabs'] as &$tab) {
+  foreach ($variables['tabs'] as $tab_id => &$tab) {
+    if ($variables['manage_mode']) {
+      // Add ui icons for manage mode.
+      $tab['ui'] = [
+        'drag' => [
+          '#type' => 'open_iconic',
+          '#size' => '15',
+          '#icon' => 'move',
+          '#color' => 'transparent',
+          '#fill' => '#000',
+          '#attributes' => [
+            'class' => ['contacts-ui-icon', 'icon-left', 'drag-handle'],
+          ],
+        ],
+        'edit' => [
+          '#type' => 'open_iconic',
+          '#size' => '15',
+          '#icon' => 'cog',
+          '#color' => 'transparent',
+          '#fill' => '#5bc0de',
+          '#attributes' => [
+            'class' => ['use-ajax', 'contacts-ui-icon', 'icon-left'],
+            'data-ajax-url' => Url::fromRoute('contacts.tab.off_canvas_form',
+              [
+                'tab' => $tab_id,
+              ]
+            )->toString(),
+            'data-dialog-type' => 'dialog',
+            'data-dialog-renderer' => 'off_canvas',
+          ],
+        ],
```

Comment:
Can we do something basic towards accessibility for these? So at least a `title` attribute and ideally some visually hidden text? Not sure how that fits into the `open_iconic` element atm?

Also, you only set `data-ajax-url`, what about if we don't have JS, do we expect this not to work?